sdk/core/Azure.Core/tests/HttpMessageTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests
+{
+    public class HttpMessageTests
+    {
+        [Test]
+        public void TryGetPropertyReturnsFalseIfNotExist()
+        {
+            HttpMessage message = new HttpMessage(new MockRequest(), new ResponseClassifier());
+
+            Assert.False(message.TryGetProperty("someName", out _));
+        }
+
+        [Test]
+        public void TryGetPropertyReturnsValueIfSet()
```

Comment:
We have the same test in PipelineTests, why a copy?

Reply:
I removed it from PipelineTests.